ic-os/bootloader/build-bootloader-tree.sh

```diff
@@ -25,11 +25,12 @@ sudo mount -t tmpfs tmpfs-podman "${TMPFS}"
 trap 'rm -rf "${TMPDIR}"; sudo podman --root "${TMPFS}" rm -f "${CONTAINER}"' exit
 TMPDIR=$(mktemp -d -t build-image-XXXXXXXXXXXX)
 
-BASE_IMAGE="docker.io/dfinity/ic-build-bazel@sha256:1978886cfda51b09057bffd60f2e5edb588c6c0b74de87696cd4e964335dba87"
+BASE_IMAGE="docker.io/ubuntu@sha256:5d070ad5f7fe63623cbb99b4fc0fd997f5591303d4b03ccce50f403957d0ddc4"
```

Comment:
Can you use [this image](https://github.com/dfinity/ic/blob/master/.github/workflows/container-mirror-images.json#L4)? Or/and maybe just try updating it to this sha256.

Reply:
How's this? It doesn't look like `ubuntu_base` is in use anymore.

Reply:
> How's this? It doesn't look like `ubuntu_base` is in use anymore.

Good point. There's no `oci.pull` rule that suggests that. Please go and just add your image. I'll clean up `ubuntu_base` latter on.